src/accessibility/check-how-accessible-your-content-si/index.md.njk

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This folder is incorrectly called: `check-how-accessible-your-content-si`

Fix the last word from "si" to "is"